site/content/docs/5.3/dark-mode.md

```diff
@@ -106,6 +106,16 @@ sitemap_exclude: true
     <div style="width: 50px; height: 50px; background-color:var(--bs-placeholder-color)" data-bs-theme="light"></div>
     <div style="width: 50px; height: 50px; background-color:var(--bs-placeholder-color)" data-bs-theme="dark"></div>
   </div>
+  <div class="d-flex flex-row gap-2 align-items-center">
+    <div style="width: 200px">--bs-gray-tweak</div>
+    <div style="width: 50px; height: 50px; background-color:var(--bs-gray-tweak)" data-bs-theme="light"></div>
+    <div style="width: 50px; height: 50px; background-color:var(--bs-gray-tweak)" data-bs-theme="dark"></div>
+  </div>
+  <div class="d-flex flex-row gap-2 align-items-center">
+    <div style="width: 200px">--bs-input-filter</div>
+    <div style="width: 50px; height: 50px; background-color:var(--bs-input-filter)" data-bs-theme="light"></div>
```

Comment:
```suggestion
    <div style="width: 50px; height: 50px; background-color: #000; filter: var(--bs-input-filter)" data-bs-theme="light"></div>
```

Reply:
Not sure we even need to display this `--bs-input-filter` in the page since it's not a color

Reply:
Discussed it together, let's remove it!